src/StatusQ/Components/StatusToolBar.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+RowLayout {
```

Comment:
Pls base this on QQC2.ToolBar

It will make it easier to properly style and extend in the future

Reply:
done

Reply:
Perfect :heart: